test/Project.toml

```diff
@@ -27,7 +27,7 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 UnPack = "3a884ed6-31ef-47d7-9d2a-63182c4928ed"
 
 [compat]
-OrdinaryDiffEq = "5.64.0"
+OrdinaryDiffEq = "~5.64.0"
```

Comment:
Can we apply this to all the environments?

Reply:
interestingly, we're using a later one with examples

Reply:
Yeah, that's strange, I thought that what we're doing was incompatible with the latest version